app/components/common/ModalGetLicenses.vue

```diff
@@ -41,6 +41,11 @@ export default Vue.extend({
     backboneDismissModal: {
       type: Boolean,
       default: false
+    },
+    showModalInitially: {
```

Comment:
This is for backwad compatibility. The old way of using this modal (and other modals) was to include them, and add a property into the containing component like `showLicensesModal` and only include the modal (using `v-if="showLicensesModal"` if the prop is `true`. 

But since I've added the [opener slot](https://github.com/codecombat/codecombat/blob/50e4912a19352568ff75287eb5a318a174ca2afc/app/components/common/ModalGetLicenses.vue#L310-L313) to the modal, it should handle its open/close state itself, and the modal should not be shown initially, just when the button in the opener slot was clicked.